src/components/wallets/AddWalletToCustomerDialog.tsx

```diff
@@ -67,7 +67,7 @@ export const AddWalletToCustomerDialog = forwardRef<DialogRef, AddWalletToCustom
               ...values,
             },
           },
-          refetchQueries: ['getCustomer'],
```

Comment:
Yep, something is not done yet but later in this feature, a new attribute will be present in the customer payload to let us know the number of active wallets customer had.
This data will help enabling/disabling the action in the top-rigth and corder dropdown in order to create a wallet 

I guess we could either decide to remove this until we have this new attribute, or keep it as it will be there anyway.

I agree tho it's not clear here.

Up for both approach !

<img width="283" alt="Monosnap ⚙️  WIP  - Credits - Create prepaid credits 2022-07-25 18-24-06" src="https://user-images.githubusercontent.com/5517077/180827497-a049c674-6d22-4db4-b825-cd4ce1d1a19f.png">

Reply:
Oh ok, whatever then :)